website/docs/r/event_orchestration_global.html.markdown

```diff
@@ -0,0 +1,159 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_event_orchestration_global"
+sidebar_current: "docs-pagerduty-resource-event-orchestration-global"
+description: |-
+  Creates and manages a Global Orchestration for an Event Orchestration.
+---
+
+# pagerduty_event_orchestration_global
+
+A [Global Orchestration](https://support.pagerduty.com/docs/event-orchestration#global-orchestrations) allows you to create a set of Event Rules. The Global Orchestration evaluates Events sent to it against each of its rules, beginning with the rules in the "start" set. When a matching rule is found, it can modify and enhance the event and can route the event to another set of rules within this Global Orchestration for further processing.
```

Comment:
This KB article URL still points to v1, which contains inaccurate information pertaining to what a Global Orchestration is in the new v2 world.

Once the v1 KB article gets replaced by the v2 article, this URL should hopefully point to the correct information.